README.md

```diff
@@ -102,7 +102,7 @@ the JS bundle. We've chosen to let Rails handle CSS, SCSS, images, fonts.
 
 ```
 cd client
-$(npm bin)/webpack -w --config webpack.rails.config.js
+npm run build:dev
 ```
 
 `client-bundle.js` is generated and saved to `app/assets/javascripts`. This is included in the
```

Comment:
By saying "yes", I meant current `npm@2.x.x`. I suggest to wait a month or so before upgrade to `3.x.x`, since there is [performance issue](https://github.com/npm/npm/issues/8826).